packages/core/src/extension.ts

```diff
@@ -128,6 +128,9 @@ export async function activate(context: vscode.ExtensionContext) {
         // TODO: Remove after some time?
         for (const conn of await Auth.instance.listConnections()) {
             if (isSsoConnection(conn) && hasScopes(conn, codeWhispererCoreScopes)) {
+                getLogger().debug(
+                    `forgetting connection: ${conn.id} with starturl/scopes: ${conn.startUrl} / ${conn.scopes}`
```

Comment:
I assume this already works good enough, but for reference, `%O` works better than toString() for printing objects
```suggestion
                getLogger().debug(
                    `forgetting connection: ${conn.id} with starturl/scopes: ${conn.startUrl} / %O`,
                    conn.scopes
```